docs/content/en/docs/getting-started/_index.md

```diff
@@ -0,0 +1,245 @@
+
+---
+title: "Getting Started"
+linkTitle: "Getting Started"
+weight: 10
+---
+
+This document showcases how to get started with Skaffold using [Docker](https://www.docker.com/)
+and Kubernetes command-line tool, [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+Aside from Docker and kubectl, Skaffold also supports a variety of other tools
+and workflows; see [How-to Guides](/docs/how-tos) and [Tutorials](/docs/tutorials) for
+more information.
+
+In this quickstart, you will:
+
+* Install Skaffold
+* Download a sample go app
+* Use `skaffold dev` to build and deploy your app every time your code changes
+* Use `skaffold run` to build and deploy your app once, on demand
+
+## Before you begin
+
+<ol>
+    <li>
+        <p><a href="https://www.docker.com/get-started">Install Docker</a></p>
+    </li>
+    <li>
+        <p><a href="https://kubernetes.io/docs/tasks/tools/install-kubectl/">Install kubectl</a></p>
+    </li>
+    <li>
+        <p>Configure kubectl to connect to a Kubernetes cluster. You can use
+        any Kubernetes platform with Skaffold; see <a href="https://kubernetes.io/docs/setup/pick-right-solution/">Picking the Right Solution</a>
+        from Kubernetes documentation for instructions on choosing the
+        right platfrom.</p>
+        <p><a href="https://cloud.google.com/kubernetes-engine/">Google Kubernetes Engine</a>
+        is a hosted Kubernetes solution. To set up kubectl with Google Kubernetes Engine,
+        see <a href="https://cloud.google.com/kubernetes-engine/docs/quickstart">Kubernetes Engine Quickstart</a>.</p>
+
+        <p><a href="https://kubernetes.io/docs/setup/minikube/">Minikube</a> is
+        a local Kubernetes solution best for development and testing. To set up
+        kubectl with Minikube, see <a href="https://kubernetes.io/docs/tasks/tools/install-minikube/">Installing Minikube</a>.</p>
+    </li>
+</ol>
+
+{{< alert title="Note" >}}
+
+If you use a non-local solution, your Docker client needs to be configured
+to push Docker images to an external Docker image registry. For setting up
+Docker with Google Container Registry, see <a href=https://cloud.google.com/container-registry/docs/quickstart>Google Container Registry Quickstart</a>.
+{{< /alert >}}
+
+## Installing Skaffold
```

Comment:
This section is empty on the website: http://35.236.7.76:1313/docs/getting-started/#installing-skaffold

Reply:
hmmm...tabs are acting up.